cmd/contour/syncer.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"fmt"
+
+	"k8s.io/client-go/tools/cache"
+)
+
+type Syncer interface {
+	HasSynced() bool
+}
+
+type SyncList struct {
```

Comment:
I'm thinking that this type should be named for what it contains, not what it does. Perhaps `InformerList` or just `Infomers` or something might be better. Then you get

func (il *InformerList) Add(I Informer)

Reply:
or maybe not, I think stand alone in this type's name is ok, but when its used in context above I find it lacking.